Dockerfile

```diff
@@ -22,13 +22,14 @@
 # https://github.com/tonistiigi/xx
 FROM --platform=$BUILDPLATFORM ghcr.io/lesnyrumcajs/xx:1.2.1 AS xx
 
-FROM --platform=$BUILDPLATFORM ubuntu:22.04 AS build-env
+FROM --platform=$BUILDPLATFORM ubuntu:24.04 AS build-env
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # install dependencies
 RUN apt-get update && \
-    apt-get install --no-install-recommends -y build-essential clang curl git ca-certificates
+    apt-get install --no-install-recommends -y build-essential clang-14 curl git ca-certificates
 RUN update-ca-certificates
+RUN ln -sf /usr/bin/clang-14 /usr/bin/clang && ln -sf /usr/bin/clang-14++ /usr/bin/clang++
```

Comment:
```suggestion
RUN ln -sf /usr/bin/clang-14 /usr/local/bin/clang && ln -sf /usr/bin/clang-14++ /usr/local/bin/clang++
```

It's nonstandard

Reply:
@aatifsyed Unfortunately this does not work.

https://github.com/ChainSafe/forest/actions/runs/10718952448/job/29721960709?pr=4731#step:9:1200
```
0.057 xx-cargo install --locked --path . --force
0.068 /usr/bin/clang not found
0.069 make: *** [Makefile:4: install-xx] Error 1`
```